charon/tests/generate-ml.rs

```diff
@@ -0,0 +1,467 @@
+#![cfg(test)]
```

Comment:
From my point of view this file is not really a test (though it also acts as a test).